src/App.js

```diff
@@ -1,78 +1,19 @@
 import "./App.css";
-import { useState } from "react";
+import List from "./components/List/list";
```

Comment:
folder names should be lowercase and component names can be CamelCase @PuniaMohit